features/step_definitions/publisher_steps.rb

```diff
@@ -0,0 +1,48 @@
+When "I go to add an artefact" do
+  click_on "Add artefact"
+  expect(page).to have_content("New artefact")
+end
+
+When "I create a draft place" do
+  fill_in "Title", with: "Smokey test draft place"
+  fill_in "Slug", with: "smokey-test-draft-place"
```

Comment:
If smokey unpublishes a page like `gov.uk/smokey-test-draft-place`, can smokey then create another artefact with the same slug? 

I think we have unique constraint on the `slug` which might require you to use a `SecureRandom.hex` suffix.

Reply:
Yep, it certainly can. It's not unpublishing as the draft is never actually published - it's simply deleting the edition of the draft, which removes the route from the draft stack ready to re-use it next time around.

Reply:
We should only ever have one of these drafts in existence at any point in time, unless it's possible for us to run multiple jobs of Smokey at once which would then have the potential for a race condition, at which point some random string might be a better bet.

Reply:
Yes I think we would want to run this smoke test in parallel (e.g. after deploys of the apps under test), so it would be great if we could find a way to enable this.